static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -103,7 +61,9 @@ const init = (): Promise<boolean> => {
     // check above is now sensitive to ad-free, it could be changed independently
     // in the future - even by accident.  Justin.
     if (!commercialFeatures.adFree) {
-        loadExternalContentWidget();
+        const edition = config.get('page.edition', '').toLowerCase();
```

Comment:
We don't use `edition` elsewhere here so I'd say we don't need to pass it as an argument, we could just declare it in `plista-outbrain-renderer` module.